cerebrium/getting-started/quickstart.mdx

```diff
@@ -6,7 +6,7 @@ description: "Get up and running with your first deployed model on Cerebrium"
 The fastest way to get started developing a Cerebrium deployment is to set up a template project using the `cerebrium init` command below. This will create a folder with all the necessary files to get you started. You can then add your code and deploy it to Cerebrium.
 
 ```bash
-cerebrium init first-project
+cerebrium init first-project --name=cerebrium-app
```

Comment:
what is "first-project" if not the name?

Reply:
the directory

Reply:
been like that for a while